research/src/utils/typography.js

```diff
@@ -43,6 +43,10 @@ const typography = new Typography({
         borderLeft: '4px solid #ccc',
       },
 
+      li: {
+        marginBottom: rhythm(0.25),
+      },
```

Comment:
Make lists less ugly

Reply:
this would make a really good commit message :)